specs/version_feature_forcast.md

```diff
@@ -24,10 +24,11 @@ Per-major version, what features would we like to have added? Note: **bug fixes
 
 ### v1.1
 
-- Autofilling Templates during character creation
+- Autofilling spells and weapons.
 - An enhanced wizard with more steps to guide people through (or a Skip button which creates an empty char)
 - Better Profile Image management/options
 - Planning for DM tools (not implementation)
+- Status modifiers for players (see: #645)
```

Comment:
Personally I think the design is pretty solid in the RFC. What do you think we could do to improve it? 

I'm also fine to move it but I think it's a really cool feature that shows off what's different about us.